plexapi/audio.py

```diff
@@ -125,6 +132,30 @@ def sync(self, bitrate, client=None, clientId=None, limit=None, title=None):
 
         return myplex.sync(sync_item, client=client, clientId=clientId)
 
+    def sonicallySimilar(
+        self: Self,
+        limit: int = 30,
+        maxDistance: float = 0.25,
+        **kwargs,
+    ) -> "list[Self]":
+        """ Find sonically similar audio items.
+
+        Parameters:
+            limit (int): maximum count of items to return, unlimited if `None`.
+            maxDistance (float): maximum distance between tracks, 0.0 - 1.0.
+            **kwargs: Additional options passed into :func:`~plexapi.base.PlexObject.fetchItems`.
+
+        Returns:
+            List[:class:`~plexapi.audio.Audio`]: list of sonically similar audio items.
+        """
+
+        key = f"/library/metadata/{self.ratingKey}/nearest?limit={limit}&maxDistance={maxDistance}"
```

Comment:
```suggestion
        key = f"{self.key}/nearest?limit={limit}&maxDistance={maxDistance}"
```

Does `limit=None` need to be excluded from the query string?

Reply:
fixed in the latest commit

- None would result in it using the server defaults which is added in the docstring